common/djangoapps/student/tasks.py

```diff
@@ -48,3 +62,224 @@ def send_activation_email(self, subject, message, from_address, dest_addr):
             exc_info=True
         )
         raise Exception
+
+
+@receiver(ENROLL_STATUS_CHANGE)
```

Comment:
I don't believe this belongs to the task.py. This receiver belongs to the signal.py